vc-defer.el

```diff
@@ -308,7 +308,7 @@ global auto reverts back on, use \\[global-auto-revert-mode]."
     ;; auto-revert-tail-mode everywhere.  These run on a timer, and so
     ;; are very intrusive in the trace output.
     (global-auto-revert-mode 0)
-    (dolist (buffer (buffer-list))
+    (dolist (_buffer (buffer-list))
```

Comment:
This can't be correct - it whould just run the below code N times for the same buffer. Probably this is missing a `with-current-buffer` instead.

Reply:
Thanks. I thought something wasn't right with this. Looks like the variable rename didn't change behaviour, but there was a minor bug which has been fixed in https://github.com/google/vc-defer/pull/4. I'll close this PR.